roles/network-multus/templates/ovs-vsctl.yml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ origin_node_provisioner_name }}
+  namespace: {{ namespace }}
+  labels:
+    tier: node
+    app: ovs-cni-plugin
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: ovs-cni-plugin
+    spec:
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: ovs-vsctl
+        command:
+        - /bin/bash
+        - -c
+        - |
+          if [ ! -f /host/usr/bin/ovs-vsctl ]; then
+            cp /usr/bin/ovs-vsctl /host/usr/bin/_ovs-vsctl
+            echo '#!/usr/bin/bash
+            _ovs-vsctl --db unix:///run/openvswitch/db.sock $@
+            ' > /host/usr/bin/ovs-vsctl
+            chmod +x /host/usr/bin/ovs-vsctl
+          fi
+          sleep infinity
+        image: {{ origin_node_provisioner_repo }}:{{ origin_node_provisioner_release }}
+        resources:
+          requests:
+            cpu: "100m"
+            memory: "50Mi"
+          limits:
+            cpu: "100m"
+            memory: "50Mi"
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: usrbin
+          mountPath: /host/usr/bin
+      volumes:
+        - name: usrbin
+          hostPath:
+            path: /usr/bin
```

Comment:
please add a new line

Reply:
What do you suggest then?

Reply:
Add a new line to the end of the file :D

Reply:
Oops, I replied to an old comment

Reply:
Done